src/pages/signup-affiliates-details/_benefits.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Flex } from '../../components/containers'
+import { Localize, localize } from 'components/localization'
+import Join from 'images/svg/signup-affiliate-details/join.svg'
+import Earn from 'images/svg/signup-affiliate-details/earn.svg'
+import Products from 'images/svg/signup-affiliate-details/products.svg'
+import Support from 'images/svg/signup-affiliate-details/support.svg'
+import device from 'themes/device.js'
+import { Header, Text } from 'components/elements'
+
+const StyledFlex = styled(Flex)`
+    @media ${device.tablet} {
+        display: none;
+    }
+`
+const Icon = styled.section`
+    width: 48px;
+    height: 64px;
+`
+const Description = styled.section`
+    padding-left: 3rem;
+`
```

Comment:
`sections` seems like a wrong semantic choice for a HTML element. `div` or `span` would be more ideal here? If `Icon` is just a wrapper, I'd suggest naming it more appropriately to it use case.

Reply:
Used `Desktop` wrapper here, and made name refactor, thanks.